Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -4041,6 +4046,11 @@ public void SetActivityGroupsExecutionStatus(BusinessFlow automateTab = null, bo
                     {
                         if (currentActivityGroup.RunStatus != eActivitiesGroupRunStatus.Passed && currentActivityGroup.RunStatus != eActivitiesGroupRunStatus.Failed && currentActivityGroup.RunStatus != eActivitiesGroupRunStatus.Stopped)
                         {
+                            if (currentActivityGroup.RunStatus == eActivitiesGroupRunStatus.Skipped)
```

Comment:
isn't better to place the IF inside Finally block of this method?

Reply:
I think this IF needs to be inside the FOR loop In order to track all the Activity Groups 
Makes sense?